components/create_post/create_post.jsx

```diff
@@ -570,45 +570,64 @@ export default class CreatePost extends React.Component {
     }
 
     handleKeyDown = (e) => {
-        if (this.props.ctrlSend && e.keyCode === KeyCodes.ENTER && e.ctrlKey === true) {
+        const postMessageKeyCombo = this.props.ctrlSend && e.keyCode === KeyCodes.ENTER && e.ctrlKey === true;
+        if (postMessageKeyCombo) {
             this.postMsgKeyPress(e);
             return;
         }
+        const ctrlOrMetaKeyPressed = e.ctrlKey || e.metaKey;
+        const messageIsEmpty = this.state.message.length === 0;
+        const draftMessageIsEmpty = this.props.draft.message.length === 0;
+        const upKeyOnly = !ctrlOrMetaKeyPressed && !e.altKey && !e.shiftKey && e.keyCode === KeyCodes.UP;
+        const shiftUpKeyCombo = !ctrlOrMetaKeyPressed && !e.altKey && e.shiftKey && e.keyCode === KeyCodes.UP;
+        const ctrlUpKeyCombo = ctrlOrMetaKeyPressed && !e.altKey && !e.shiftKey;
```

Comment:
Consider removing `Up` from the name of this variable, since it's used in combination with both the up and down arrow keys on lines 589 and 591 respectively.

Reply:
Great catch. Will update